app/assets/javascripts/app/components/button/button.js

```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```

Comment:
Shouldn't you bind to controller so we can avoid using the scope? We are doing this on the other directives as well.